src/registry.json

```diff
@@ -2078,6 +2087,67 @@
         }
       }
     },
+    "npm:@polkagate/snap": {
+      "id": "npm:@polkagate/snap",
+      "metadata": {
+        "name": "PolkaGate",
+        "author": {
+          "name": "PolkaGate",
+          "website": "https://polkagate.xyz/"
+        },
+        "website": "https://apps.polkagate.xyz/",
+        "summary": "Unlock the Power of Polkadot. Seamlessly manage accounts, connect to Polkadot dApps, and sign transactions.",
```

Comment:
Regardless of how the developer writes "dapp," we always spell it "dapp."

Reply:
Done.